sbt_common/messaging/src/main/scala/uk/ac/wellcome/messaging/sqs/SQSStream.scala

```diff
@@ -67,10 +67,6 @@ class SQSStream[T] @Inject()(actorSystem: ActorSystem,
         logger.error(
           s"Unrecognised failure while processing message ${message.getMessageId}",
           exception)
-        metricsSender.incrementCount(
-          metricName = s"${streamName}_MessageProcessingFailure",
-          count = 1.0
-        )
```

Comment:
Okay, maybe we should keep this, oops.

Reply:
If this goes away, We can get rid of GracefulFailureException cause this is the only difference in behaviour between the two types of exception

Reply:
Ah no, it's covered above.

Reply:
There’s also the difference in logging – a GracefulFailureException logs a WARN and a message, whereas an unrecognised exception includes the full stack trace for extra debugging.